tt1-blocks/functions.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * Functions and definitions
+ *
+ * @link https://developer.wordpress.org/themes/basics/theme-functions/
+ *
+ * @package TT1_Blocks
+ * @since 0.1
+ */
+
+if ( ! function_exists( 'tt1_blocks_setup' ) ) {
+	/**
+	 * Sets up theme defaults and registers support for various WordPress features.
+	 *
+	 * Note that this function is hooked into the after_setup_theme hook, which
+	 * runs before the init hook. The init hook is too late for some features, such
+	 * as indicating support for post thumbnails.
+	 *
+	 * @since 0.1
+	 *
+	 * @return void
+	 */
+	function tt1_blocks_setup() {
+		/*
+		 * Make theme available for translation.
+		 * Translations can be filed in the /languages/ directory.
+		 * If you're building a theme based on TT1 Blocks, use a find and replace
+		 * to change 'tt1-blocks' to the name of your theme in all the template files.
+		 */
+		load_theme_textdomain( 'tt1-blocks', get_template_directory() . '/languages' );
+
+		// Add default posts and comments RSS feed links to head.
+		add_theme_support( 'automatic-feed-links' );
+
+		/*
+		 * Enable support for Post Thumbnails on posts and pages.
+		 *
+		 * @link https://developer.wordpress.org/themes/functionality/featured-images-post-thumbnails/
+		 */
+		add_theme_support( 'post-thumbnails' );
+		set_post_thumbnail_size( 1568, 9999 );
+
+		// Add support for Block Styles.
+		add_theme_support( 'wp-block-styles' );
+
+		// Add support for editor styles.
+		add_theme_support( 'editor-styles' );
+
+		// Enqueue editor styles.
+		add_editor_style(
+			array(
+				'./assets/css/blocks.css',
+				'./assets/css/style-shared.css',
+			)
+		);
+
+		// Add support for responsive embedded content.
+		add_theme_support( 'responsive-embeds' );
+	}
+}
+add_action( 'after_setup_theme', 'tt1_blocks_setup' );
+
+/**
+ * Enqueue scripts and styles.
+ *
+ * @since 0.1
+ *
+ * @return void
+ */
+function tt1_blocks_enqueue_styles() {
+	wp_enqueue_style( 'tt1-blocks-style', get_template_directory_uri() . '/style.css', array(), wp_get_theme()->get( 'Version' ) );
+	wp_enqueue_style( 'tt1-blocks-style-shared', get_template_directory_uri() . '/assets/css/style-shared.css', array(), wp_get_theme()->get( 'Version' ) );
+	wp_enqueue_style( 'tt1-blocks-blocks', get_template_directory_uri() . '/assets/css/blocks.css', array(), wp_get_theme()->get( 'Version' ) );
+}
+add_action( 'wp_enqueue_scripts', 'tt1_blocks_enqueue_styles' );
+
+/**
+ * Enqueue block editor script.
+ *
+ * @since 0.1
+ *
+ * @return void
+ */
+function tt1_blocks_block_editor_script() {
+
+	wp_enqueue_script( 'tt1-blocks-unregister-block-style', get_theme_file_uri( '/assets/js/unregister-block-style.js' ), array( 'wp-blocks', 'wp-dom' ), wp_get_theme()->get( 'Version' ), true );
+}
+add_action( 'enqueue_block_editor_assets', 'tt1_blocks_block_editor_script' );
+
+// Block Styles.
+require get_template_directory() . '/inc/block-styles.php';
```

Comment:
I think everything in this file except the style shared/blocks stylesheets and the block styles stuff can go, don't you think @carolinan ? 

We can also discuss if those stylesheets can be consolidated into one if they end up being necessary at all